include/wx/persist/dataview.h

```diff
@@ -94,6 +95,7 @@ class wxPersistentDataViewCtrl : public wxPersistentWindow<wxDataViewCtrl>
 
     virtual bool Restore() override
     {
+        std::map<int, wxDataViewColumn*> order;
```

Comment:
IMO the use of `map` here is confusing because we never actually use its key. Why not just use `std::vector<wxDataViewColumn*>` instead.

Reply:
Map is required for implicit sorting, there's no wxMap. Vector give you unsorted sequence during the iteration or it must be sorted.